source/adios2/engine/ssc/SscWriter.cpp

```diff
@@ -34,6 +36,22 @@ SscWriter::SscWriter(IO &io, const std::string &name, const Mode mode,
     m_GlobalWritePattern.resize(m_WriterSize);
     m_GlobalReadPattern.resize(m_ReaderSize);
 
+    /*
```

Comment:
Can these be removed?

Reply:
> Can these be removed?

All the redundant comments have been removed.